src/main/java/uk/gov/hmcts/darts/audio/service/impl/MediaRequestServiceImpl.java

```diff
@@ -535,11 +535,11 @@ public MediaHideResponse adminHideOrShowMediaById(Integer mediaId, MediaHideRequ
 
                 objectAdminActionEntity = objectAdminActionRepository.saveAndFlush(objectAdminActionEntity);
 
-                response = AdminMediaSearchResponseMapper.mapHideOrShowResponse(mediaEntity, objectAdminActionEntity);
+                response = GetAdminMediaResponseMapper.mapHideOrShowResponse(mediaEntity, objectAdminActionEntity);
```

Comment:
For consistency Admin should be at the front

Reply:
As discussed, i wanted to keep 'Admin' and 'Media' together, as thats whats in the endpoint url:-
GET /admin/media

OpenapiGenterator will generate it like this:-
AdminMediaGet

so i can change it to that if it makes more sense